docs/classes/tickets.html

```diff
@@ -1031,8 +1031,8 @@ <h3><span class="tsd-flag ts-flagStatic">Static</span> return<wbr>ToQueue</h3>
 								</div>
 								<p>Returning a ticket to the queue makes the ticket&#39;s status go back to &#39;NEW&#39;, a Ticket
 									Created event will fire, and the ticket will appear back in the queue. The ticket can be
-									ordered to be first in queue, somewhere in the middle, or the last in queue, depending on
-								the reason the visitor needs to go back to the queue and when they will be back for service.</p>
+									ordered to be first in queue or the last in queue, depending on the reason
+									the visitor needs to go back to the queue and when they will be back for service.</p>
```

Comment:
By the way, it's not correct, the Ticket Created event will not be fired. It fires ticket changed and ticket reordered events.

Reply:
created issue https://github.com/Qminder/javascript-api/issues/378

Reply:
Come on 😀 Let's fix it in this PR